setup.py

```diff
@@ -21,7 +21,8 @@
 REQS_BASE = [
     'bunch>=1.0.1',
     'six>=1.10.0',
-    'retrying>=1.3.3'
+    'retrying>=1.3.3',
+    'oslo.i18n>=3.15.3'
```

Comment:
@baizhang @chiniforooshan @mbookman Just wondering what this is currently used for?